build.gradle

```diff
@@ -36,6 +36,7 @@ dependencies {
 appengine {  // App Engine tasks configuration
   run {      // local (dev_appserver) configuration (standard environments only)
     port = 8080                 // default
+    host = "0.0.0.0"
```

Comment:
Need to bind "any IP" to make sure that port-forwarding from outside the container can connect to the server running inside.